lib/beam-calls-data.tsx

```diff
@@ -72,8 +72,8 @@ export const beamCallsData: BeamCall[] = [
         type: "slides"
       },
       {
-        title: "Alex's slides on post-quantum distributed validators",
-        url: "./resources/beam-calls/status-hash-sig.pdf",
+        title: "Alex's slides on post-quantum Distributed Validators",
```

Comment:
Is "Distributed Validator" a specific term that requires capitalization?

Reply:
Doesn't have to be! I still feel like i'm trying to make DVs a 'thing', but if they're so well understood by now that they don't need to sound like a noun that's fine by me, and indicative of progress :) lowercased them all now.